.github/workflows/release.yml

```diff
@@ -1,58 +1,73 @@
+      - 'graphs/**'
+      - 'sandbox/**'
+
+  workflow_dispatch:
     inputs:
-      timeout-minutes:
-        description: 'Timeout in minutes for the release job'
-        type: number
-        default: 45
-      # Release-specific inputs
-      target-version:
-        description: 'Target version to release (for latest releases)'
+      targetVersion:
+        description: 'Version to publish on latest tag (e.g. 5.0.0)'
         type: string
-        required: false
-      skip-ecosystem-tests-checks:
+      skipEcosystemTestsChecks:
         description: 'Skip ecosystem tests checks'
         type: boolean
-        default: false
-      skip-packages:
-        description: 'Skip publishing some packages'
+      skipPackages:
+        description: 'Skip publishing some packages? (e.g. `@prisma/debug,@prisma/internals`)'
         type: string
-        required: false
-      only-packages:
-        description: 'Only publish some packages'
+      onlyPackages:
+        description: 'Only publish some packages? (e.g. `@prisma/debug,@prisma/internals`)'
         type: string
-        required: false
-      dry-run:
-        description: 'Perform a dry run without publishing'
+      dryRun:
+        description: 'Check to do a dry run (does not publish packages)'
         type: boolean
-        default: false
-      force-integration-release:
-        description: 'Force an integration release'
+      forceIntegrationRelease:
+        description: 'Check to force an integration release for any given branch name'
         type: boolean
-        default: false
-      # Notification settings
-      include-failure-notifications:
-        description: 'Include failure notification job'
-        type: boolean
-        default: false
-      failure-title-template:
-        description: 'Template for failure notification title'
-        type: string
-        default: 'prisma/prisma Release failed :x:'
 
 env:
   # To hide "Update available 0.0.0 -> x.x.x"
   PRISMA_HIDE_UPDATE_MESSAGE: 'true'
+  # Environment variables based on trigger type and parameters
+  RELEASE_VERSION: ${{ github.event.inputs.targetVersion }}
+  SKIP_ECOSYSTEMTESTS_CHECK: ${{ github.event.inputs.skipEcosystemTestsChecks || '' }}
+  SKIP_PACKAGES: ${{ github.event.inputs.skipPackages }}
+  ONLY_PACKAGES: ${{ github.event.inputs.onlyPackages }}
+  DRY_RUN: ${{ github.event.inputs.dryRun || '' }}
+  FORCE_INTEGRATION_RELEASE: ${{ inputs.forceIntegrationRelease || '' }}
+  INCLUDE_FAILURE_NOTIFICATIONS: ${{ github.event.inputs.targetVersion && 'true' || '' }}
+  FAILURE_TITLE_TEMPLATE: >-
+    ${{
+      github.event.inputs.targetVersion &&
+      format('prisma/prisma Release {0} from {1} on latest tag failed :x:', github.event.inputs.targetVersion, github.ref_name) ||
+      'prisma/prisma Release failed :x:'
+    }}
 
 jobs:
   release:
-    timeout-minutes: ${{ inputs.timeout-minutes }}
+    timeout-minutes: ${{ github.event.inputs.targetVersion && 45 || 75 }}
```

Comment:
If I were to guess, they probably diverged at some point inadvertently. I'm fine with it either way though.